src/pages/learn/serving-over-http.mdx

```diff
@@ -34,37 +77,63 @@ http://myapi/graphql?query={me{name}}
 
 Query variables can be sent as a JSON-encoded string in an additional query parameter called `variables`. If the query contains several named operations, an `operationName` query parameter can be used to control which one should be executed.
 
-### `POST` request
+#### Choosing an HTTP method
 
-A standard GraphQL POST request should use the `application/json` content type, and include a JSON-encoded body of the following form:
+When choosing an HTTP method for a GraphQL request, there are a few points to consider. First, support for HTTP methods other than `POST` will be at the discretion of the GraphQL server, so a client will be limited to the supported verbs.
 
-```json
-{
-  "query": "...",
-  "operationName": "...",
-  "variables": { "myVariable": "someValue", ... }
-}
-```
+Additionally, the `GET` HTTP method may only be used for query operations, so if a client sends a mutation operation in a document then it must use the `POST` method instead.
+
+When servers that do support the `GET` method for query operations, a client may be encouraged to leverage this option to facilitate HTTP caching or edge caching in a content delivery network (CDN). However, because GraphQL documents strings may be quite long for complex operations, the query parameters may exceed the limits that browsers and CDNs impose on URL lengths.
+
+In these cases, it would be necessary to use something like _trusted documents_ (sometimes referred to as _persisted queries_) or _automatic persisted queries_, where the server stores hashes of known GraphQL documents and the clients can send a hash as the `query` parameter value in a `GET` request instead of the full source text of the query operation.
```

Comment:
This network optimization doesn't require _trusted_ documents: _persisted_ documents is sufficient for this. Different techniques supply the identifier in different ways (the persisted documents RFC is trying to standardize this) so I've avoided being specific here though I have linked to the RFC:

```suggestion
In these cases, a common approach is for the server to store identified GraphQL documents using a technique such as [_persisted documents_](https://github.com/graphql/graphql-over-http/pull/264), _automatic persisted queries_, or _trusted documents_. This allows the client to send a short document identifier instead of the full query text.
```